lib/Server.js

```diff
@@ -243,6 +243,11 @@ function Server (compiler, options = {}, _log) {
     res.end('</body></html>');
   });
 
+  app.get('/invalidate', (req, res) => {
```

Comment:
Do you need func params here?

Reply:
I need `res`, but `req` can be removed. Done :)